timesketch/lib/analyzers/base_sessionizer_test.py

```diff
@@ -172,44 +168,37 @@ def _create_mock_event(event_id, quantity, time_diffs=None,
         time_diffs.extend([time_diffs[len(time_diffs) - 1]] *
                           (quantity - len(time_diffs)))
 
-    # Setup for Event object initialisation
```

Comment:
Since a new MockDataStore intsance is not used and _create_mock_event is not used as a Event object generator, the code needed to create them is removed.